config/SGGame/items.json

```diff
@@ -707,6 +707,215 @@
             "triggerEvo": true
         }
     },
+     "abomasite": {
+        "id": "abomasite",
+        "name": "Abomasite",
+        "slot": "items",
+        "use": {
+            "battleOnly": true
+        }
```

Comment:
Mega stones cant be "used" from the bag like potions or pokeballs, only given to pokemon and used in battle.

So that means theres no need for a `battleOnly` tag, and since theres nothing else in `use` theres no need for a `use` section.

Reply:
Without battleOnly it crashes when someone click on use instead of give.

Reply:
That would be an issue with the bag system then, since there are numerous berries that are setup like I mentioned above. I'll add it to the todo list.

Reply:
Oh okay.